lib/opentok/streams.rb

```diff
@@ -75,5 +75,35 @@ def layout(session_id, opts)
       (200..300).include? response.code
     end
 
+    # Force a specific stream connected to an OpenTok session to mute itself.
+    #
+    # @param [String] session_id The session ID of the OpenTok session.
+    # @param [String] stream_id The stream ID of the stream in the session.
+    #
+    def force_mute(session_id, stream_id)
+      response = @client.force_mute_stream(session_id, stream_id)
+    end
+
+    # Force all streams connected to an OpenTok session to mute themselves.
+    #
+    # @param [String] session_id The session ID of the OpenTok session.
+    # @param [Hash] opts An optional hash defining options for muting action. For example:
+    # @option opts [true, false] :active Whether streams published after this call, in
+    # addition to the current streams in the session, should be muted (true) or not (false).
+    # @option opts [Array] :excluded_streams The stream IDs for streams that should not be muted.
+    # This is an optional property. If you omit this property, all streams in the session will be muted.
+    # @example
+    # {
+    #   "active": true,
+    #   "excluded_streams": [
+    #     "excludedStreamId1",
+    #     "excludedStreamId2"
+    #   ]
+    # }
+    #
+    def force_mute_all(session_id, opts = {})
```

Comment:
I sort of think the `force_mute_all()` method should not be in the Streams class. It applies to all streams in a session, not to a specific stream. But i don't know where else to put it. We don't have an OpenTok::Sessions object. Wdyt, @superchilled? (I am not very proficient in Ruby, and if you think it's OK to have this method here, that's fine.)